dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/Ds3PutJob.java

```diff
@@ -167,20 +169,23 @@ public void executeJob() throws Exception {
 
         job.transfer(file -> FileChannel.open(PathUtil.resolveForSymbolic(fileMapper.get(file)), StandardOpenOption.READ));
 
-        waitForPermanentStorageTransfer(totalJobSize)
-                    .observeOn(JavaFxScheduler.platform())
-                    .doOnComplete(() -> {
-                        LOG.info("Job transferred to permanent storage location");
-
-                        final String newDate = dateTimeUtils.nowAsString();
-                        loggingService.logMessage(
-                                StringBuilderUtil.jobSuccessfullyTransferredString(PUT,
-                                        FileSizeFormat.getFileSizeType(totalJobSize), bucket + "\\" + targetDir, newDate,
-                                        resourceBundle.getString("permanentStorageLocation"), false).toString(), LogType.SUCCESS);
-
-                        Ds3PanelService.refresh(remoteDestination);
-                        ParseJobInterruptionMap.removeJobID(jobInterruptionStore, this.getJobId().toString(), ds3Client.getConnectionDetails().getEndpoint(), deepStorageBrowserPresenter, loggingService);
-                    }).subscribe();
+        final Disposable d = waitForPermanentStorageTransfer(totalJobSize)
+                .observeOn(JavaFxScheduler.platform())
+                .doOnComplete(() -> {
+                    LOG.info("Job transferred to permanent storage location");
+
+                    final String newDate = dateTimeUtils.nowAsString();
+                    loggingService.logMessage(
+                            StringBuilderUtil.jobSuccessfullyTransferredString(PUT,
+                                    FileSizeFormat.getFileSizeType(totalJobSize), bucket + "\\" + targetDir, newDate,
+                                    resourceBundle.getString("permanentStorageLocation"), false).toString(), LogType.SUCCESS);
+
+                    Ds3PanelService.refresh(remoteDestination);
+                    ParseJobInterruptionMap.removeJobID(jobInterruptionStore, this.getJobId().toString(), ds3Client.getConnectionDetails().getEndpoint(), deepStorageBrowserPresenter, loggingService);
+                }).subscribe();
+        while(!d.isDisposed()) {
+            Thread.sleep(100);
```

Comment:
We probably should increase this sleep.  This is quite a quick sleep, and chances are we don't need this level of granularity.  Try increasing it to once every 500 millis or something like that.  Once a second may also work just fine.